modules/core/arrow-core-data/src/main/kotlin/arrow/typeclasses/Apply.kt

```diff
@@ -78,6 +118,20 @@ interface Apply<F> : Functor<F> {
   ): Kind<F, Z> =
     a.product(b).product(c).product(d).product(e).map(lbd)
 
+  fun <A, B, C, D, E, Z> mapN(
+    a: Kind<F, A>,
+    b: Kind<F, B>,
+    c: Kind<F, C>,
+    d: Kind<F, D>,
+    e: Kind<F, E>,
+    lbd: (Tuple5<A, B, C, D, E>) -> Z
+  ): Kind<F, Z> =
+    a.product(b).product(c).product(d).product(e).map(lbd)
+
+  @Deprecated(
+    "map is being renamed to mapN",
+    ReplaceWith("mapN(a, b, c, d, e, ff, lbd)")
```

Comment:
This and all below deprecation notices for `mapN` reference `ff`, but the variable is called `f` in all cases